websites/H/HentaiZ/dist/metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+  "author": {
+    "name": "Kinue",
+    "id": "634246091293851649"
+  },
+  "service": "HentaiZ",
+  "description": {
+    "vi_VN": "HentaiZ - Trang xem hentai anime vietsub online miễn phí chất lượng cao",
+    "en": "HentaiZ - Top #1 Anime Hentai website in Vietnam with high quality"
+  },
+  "url": [
+    "hentaiz.net",
+    "www.hentaiz.net"
+  ],
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/685u3IG.png",
+  "thumbnail": "https://i.imgur.com/rLkSLvZ.png",
```

Comment:
Nope, like I said, homepage must be on thumbnail and just blur images?

Reply:
kk hold on